packages/webview_flutter/example/lib/main.dart

```diff
@@ -82,6 +82,10 @@ class _WebViewExampleState extends State<WebViewExample> {
           onPageFinished: (String url) {
             print('Page finished loading: $url');
           },
+          onPullToRefresh: () {
+            print('Pull to refresh.');
+            return Future.delayed(Duration(seconds: 1));
```

Comment:
is it common that folks would want to reload the page?

I don't have the exact data, but I suspect that is the main use case.
It might be worth to add the common use case to the example.

You never know if you encounter a bug  :)